doc/releases/release-notes-3.5.rst

```diff
@@ -100,14 +100,132 @@ Bluetooth
 
```

Comment:
Any chance you could have a couple bullet point with high-level summary (thinking 3-5 entries) of what are the most notable changes across all bluetooth? It's kind of difficult to figure it out by looking at the pretty large/flat list of bullet items.

Reply:
I have added couple lines now under Controller.

@Thalley @jori-nordic @PavelVPV may be you can give some lines and I will include them...

Reply:
For LE audio you can add something like

Improved memory usage of codec configurations codec capabilities. Fixed several bugs in BAP and the BAP-related services (ASCS, PACS, BASS), as well as missing features such as proper notification handling.